pkg/test/frameworkSettings/netcoreapp5.0/settings.targets

```diff
@@ -0,0 +1,33 @@
+<Project>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
TargetFramework can be removed as it is redundant.  This targets file doesn't even load unless TFM is already NETCoreApp5.0.